website/source/api/agent/service.html.md

```diff
@@ -424,6 +424,10 @@ The table below shows this endpoint's support for
 | ---------------- | ----------------- | ------------- | --------------- |
 | `NO`             | `none`            | `none`        | `service:write` |
 
+### Query string parameters
+
+- `replace-existing-checks` - Missing healthchecks from the request will be deleted from the agent. Using this parameter allows to idempotently register a service and its checks whithout having to manually deregister checks.
```

Comment:
I think it would be useful to add the parameter to the sample request in L566

Reply:
I agree, especially since IMHO this should be the default IMHO in future versions. What do you think @i0rek  ?

Reply:
👍 on adding to example. Hans is away this week but I think if we can add that example this is good to merge!

I'm going to approve anyway but would be great to have that added